packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -79,7 +79,7 @@ class Autofill extends BaseComponent<IAutofillProps, IAutofillState>, implements
   readonly cursorLocation: number | null;
   // (undocumented)
   static defaultProps: {
-    enableAutofillOnKeyPress: KeyCodes[];
+    enableAutofillOnKeyPress: (38 | 40)[];

```

Comment:
This looks really wrong here.. it would be really helpful to understand why API Extractor says 38 | 40. @pgonzal , is this an API Extractor bug?

Reply:
perhaps d.ts actually resolved to actual values, since KeyCodes is now a type alias...

Reply:
typescript struggles to widen the type because it thinks these are never assigned any other types probably. I can coerce this to be more suitable.